docs/src/pages/guides/aliases.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Aliases
+---
+
+Aliasing is an integral part when it comes to clean and readable code, that's why Astro comes with support for custom aliases out of the box.
+
+## What are aliases?
+
+Aliases are _keywords_ you can use instead of paths. 
+So instead of `../../components/SomeComponent/` you can use `components/SomeComponent`. Cool, right!
+
+## Adding a custom alias
+
+If you navigate to the root directory of your project and open the `snowpack.config.mjs` file you will notice that there are some pre-defined aliases already.
```

Comment:
To add custom aliases to your project, located in the root of your project is the `snowpack.config.mjs` file. This configuration file contains the instructions for Astro's build tool, [Snowpack](www.snowpack.dev), on how to build out your Astro project.  Inside the file you will notice that there are already some predefined aliases.

Reply:
note that some projects don't have this file! mention to create one if you don't have it yet.